chef/data_bags/crowbar/bc-template-nova.json

```diff
@@ -31,8 +31,7 @@
         "database": "nova"
       },
       "rbd": {
-        "user": "",
-        "secret_uuid": ""
+        "user": ""
```

Comment:
No, this has to stay empty if we would like to use multi ceph backend for cinder. I think this has to considered for ephemeral rbd support in Nova.